### ISSUE TITLE
FormReader reads wrong mimetype from stream

### ISSUE THREAD

**Issue by julianpoemp** [dmitriy-nz/nestjs-form-data#37]
The `loadFile()` function in FormReader.ts reads a wrong mime type from a stream. In my case it's `audio/vnd.wave` instead of `audio/wave`. That leads to a failing `HasMimeType` validation.

![Bildschirmfoto 2022-09-16 um 14 21 27](https://user-images.githubusercontent.com/25926588/190638206-aa6663ae-fa15-4405-b793-308aee7fb7cd.png)

I assume it's a bug from the `file-type` package. You are using an old version, v16.5.4, perhaps this issue is solved if you upgrade to the latest version?

I tested it uploading a file using FileSystemStorage and a DTO that uses the `HasMimeType`decorator. I'm currently using nestjs-form-data@1.8.3 and NestJS@9.0.11 with platform-express@9.0.11

As a workaround I downgraded back to nestjs-form-data@1.7.1.

**Comment by dmitriy-nz** [comment]
Hi! Can you confirm that the new version detects the correct mime type for your file?

**Comment by julianpoemp** [comment]
@dmitriy-nz can't create a custom build of nestjs-form-data with the latest file-type package. Building with `npm run build` doesn't create a valid package. The build can't be installed via `npm link` or as archive in my project.

**Comment by julianpoemp** [comment]
@dmitriy-nz found a way to create valid packages (using `npm pack`).

There is a huge ESM problem with file-type imports in commonjs packes so I can't use the latest version because I get the error as described [here](https://github.com/sindresorhus/file-type/issues/535) . There are a lot of issues discussed in file-type where a lot of people don't recommend to update it as long as you are using commonjs. I think this will be an issue in the future.

Furthermore I found out that the "correct" mime types for audio/wave and audio/x-wave should be `audio/vnd.wave` according to this [PR](https://github.com/sindresorhus/file-type/pull/150). So I changed the valid mime types an my side and now it works.
